package.json

```diff
@@ -142,6 +142,7 @@
     "gulp-util": "3.0.8",
     "html-loader": "0.5.1",
     "html-webpack-plugin": "2.30.1",
+    "each-props": "^1.3.0",
```

Comment:
Why do we need this?

Reply:
Leftover from trying to fix the build. Removed.